pkg/reconcile/pipeline/context/impl.go

```diff
@@ -120,51 +120,61 @@ func (i *impl) Services() ([]pipeline.Service, error) {
 	return services, nil
 }
 
+// Application return a list of applications.
+// And if no application found, return an error
```

Comment:
please update  the interface doc in`pkg/reconcile/pipeline/api.go`

Reply:
Done. 34cd1e9bf29f243191f69e5352b0440b288b3927